t/info.t

```diff
@@ -147,6 +147,36 @@ is_json_GET(
   action_bad_regex('/info/biotypes/wibble', qr/Could not fetch adaptor for species .+/, 'Bogus species means error message');
 }
 
+# /info/biotypes/groups
+{
+  my $biotypes_json = json_GET('/info/biotypes/groups', 'Get the list biotype groups');
+  is(ref($biotypes_json), 'ARRAY', 'Array wanted from endpoint');
+  cmp_ok(scalar(@{$biotypes_json}), '==', 8, 'Ensuring we have the right number of biotype groups');
```

Comment:
A weak test. Perhaps you could sort and spot-check the string value of the first or last element?

Reply:
indeed, will do that